src/KafkaFlow.Retry.SqlServer/RetryDurableDefinitionBuilderExtension.cs

```diff
@@ -5,18 +5,28 @@ public static class RetryDurableDefinitionBuilderExtension
         public static RetryDurableDefinitionBuilder WithSqlServerDataProvider(
             this RetryDurableDefinitionBuilder retryDurableDefinitionBuilder,
             string connectionString,
-            string databaseName)
+            string databaseName,
+            string schema)
```

Comment:
Instead of a default value on the parameter, create an overload of `WithSqlServerDataProvider(...)` that assumes `schema` to be `"dbo"`.

Reply:
In fact, @carlosgoias actually did create an overload starting line 24 👌

@carlosgoias instead of passing `null` to the parameter, pass `"dbo"` and we should have accomplished the objective.